bower.json

```diff
@@ -3,8 +3,8 @@
   "version": "v2.0.0-beta.1",
```

Comment:
lets get rid of the `bower.json` version (see: Esri/esri-leaflet#829 for more info)